docs/extensibility/payment-method-integration.md

```diff
@@ -152,3 +154,31 @@ This contains various details about the payment result returned to the client an
 -   set the status to return for the payment method (one of `success`, `failure`, `pending`, `error`).
 -   set a redirect url.
 -   set any additional payment details (in case you need to return something for your client to further process with).
+
+### Putting it all together
```

Comment:
I :heart: the way you've broken this section down into an explanation of steps followed by an example of the result.